plot.py

```diff
@@ -0,0 +1,37 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+
+# df = pd.read_csv("test_for_plot.txt", sep="\t")
+# df['mean_rain'] = [0.0, 0.0, 0.0, 0.0, 0.0, 2.0, 11.0, 20.0, 17.0,
+#                    6.0, 0.5, 0.2, 0.2, 0.0, 0.0, 0.0, 0.0, 0.0,
+#                    0.0, 0.0, 0.0, 0.2, 0.1, 0.2]
+
+
+def create_plot(df, parameters):
```

Comment:
What's a `df`? I should know that by just taking a glance at the code I'm reading.

Reply:
Done. Changed to full_forecast_dataframe.